Configuring for pure-python-without-pypy

sallner opened the pull request:
In this iteration the required coverage can be configured in the `.meta.cfg`.

sallner committed:
Configuring for pure-python-without-pypy


Configuring for pure-python-without-pypy

dataflake reviewed (commented):
I don't know anything about or use this particular package and have not had the time to follow the work on the configuration generator, so I can't say much about it.

mgedmin reviewed (approved):
LGTM

sallner committed:
Configuring for pure-python-without-pypy

Configuring for pure-python-without-pypy

Configuring for pure-python-without-pypy
